libcontainer/rootfs_linux.go

```diff
@@ -776,7 +776,7 @@ func mountPropagate(m *configs.Mount, rootfs string, mountLabel string) error {
 	}
 
 	if err := syscall.Mount(m.Source, dest, m.Device, uintptr(flags), data); err != nil {
-		return err
+		return fmt.Errorf("Dan mountPropagate Failed %s %s %d %s %v", m.Source, dest, flags, data, err)
```

Comment:
@rhatdan If you're going to do that, please consider using https://github.com/pkg/errors to revamp all error handling in runC. It would be a fairly big task, but the result will be that we get full error wrapping and stack traces *FOR FREE*.